internal/commands/scan.go

```diff
@@ -811,6 +861,23 @@ func setupScanTypeProjectAndConfig(
 	return err
 }
 
+func getApplication(applicationName string, applicationsWrapper wrappers.ApplicationsWrapper) (*wrappers.Application, error) {
+	if applicationName != "" {
+		params := make(map[string]string)
+		params["name"] = applicationName
+		resp, _, err := applicationsWrapper.Get(params)
+		if err != nil {
+
+			return nil, err
+		}
+		if resp.Applications != nil && len(resp.Applications) > 0 {
+			application := resp.Applications[0]
+			return &application, nil
```

Comment:
why using & here?

Reply:
I had to use reference here because otherwise it's not possible to return nil in line 879 (and I didn't want to return an empty object instead).
for that reason I also had to return the reference to the application in life 875